vertical-pod-autoscaler/samples/vpa.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: poc.autoscaling.k8s.io/v1alpha1
+kind: VerticalPodAutoscaler
+metadata:
+  name: sample-vpa
+spec:
+  selector:
+    matchLabels:
+      autoscaling: enabled
```

Comment:
We don't want to promote super generic labels. Ideally each deployment should have its own VPA. Please add other labels here (and reduce their number - no need for 3).